scripts/deploy.sh

```diff
@@ -69,11 +76,13 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
                      awk '/d[b-z]+ disk 0/{ if ($4>m && $4>='$DATA_DISK_MIN_SIZE'){m=$4; d=$1}}; END{print d}')
   # Only set the secondary disk device option if there is one
   if [ -n "${DATA_DISK_DEVICE}" ]; then
-    export BOOTSTRAP_OPTS="${BOOTSTRAP_OPTS} bootstrap_host_data_disk_device=${DATA_DISK_DEVICE}"
+    export BOOTSTRAP_OPTS="${BOOTSTRAP_OPTS} -e bootstrap_host_data_disk_device=${DATA_DISK_DEVICE}"
   fi
   # force the deployment of haproxy for an AIO
   export DEPLOY_HAPROXY="yes"
   if [[ ! -d /etc/openstack_deploy/ ]]; then
+    # bootstrap-aio.sh will fail if pip10 is present so lets remove it and
+    # replace it with the versions from global-requirement-pins.txt
```

Comment:
Is this the correct location for the comments here?  It's adding comments for no changes to this section of code.